virt/logging_events_monitoring/virt-monitoring-vm-health.adoc

```diff
@@ -5,10 +5,15 @@ include::modules/virt-document-attributes.adoc[]
 
 toc::[]
 
-Use this procedure to create liveness and readiness probes
-to monitor virtual machine health.
+A virtual machine instance (VMI) can become unhealthy due to transient issues such as temporary connectivity loss, deadlocks, or problems with external dependencies. A health check periodically performs diagnostics on a VMI by using a combination of the readiness and liveness probes.
```

Comment:
I don't think you need "temporary" here, since "transient" applies to the whole list.

I am confused by this sentence: "A health check periodically performs diagnostics on a VMI by using a combination of the readiness and liveness probes." 
- Is a 'health check' in this context something other than what you configure in this assembly? 
- "...by using a combination of the readiness and liveness probes" makes it sound like you have to use both, but the assembly makes it seem like you can use one or the other.

Reply:
Removed "temporary" 

A "health check" means the same thing as configuring the probes as described in the assembly. I changed the second sentence to "by using any combination of the readiness and liveness probes".